x/stakeibc/keeper/unbonding_records_get_host_zone_unbondings_msgs_test.go

```diff
@@ -121,13 +128,16 @@ func (s *KeeperTestSuite) TestGetHostZoneUnbondingMsgs_Successful() {
 	// verify the callback attributes are as expected
 	actualCallbackResult, err := s.App.StakeibcKeeper.UnmarshalUndelegateCallbackArgs(s.Ctx, actualCallbackArgs)
 	s.Require().NoError(err, "could unmarshal undelegation callback args")
-	s.Require().Equal(len(tc.hostZone.Validators), len(actualCallbackResult.SplitDelegations), "number of split delegations in success unbonding case")
+
+	// The zero weight validator should not be included in the message
+	expectedNumMessages := len(tc.hostZone.Validators) - 1
```

Comment:
Nice! 

Just to cover all our bases, could you please add a test for the case where we only have 0-weight validators? It should send no messages in that case I believe

Reply:
This is not what I have added FYI - I think someone else added this. :)

Reply:
It's from #612 @riley-stride

Reply:
@sampocs should we add this test in #612, or add it here (per campfire rule) ?

Reply:
This is just the cleanup unbonding records function - it doesn't send any messages and is pretty simple. I personally don't think the 0-weight only case adds much. Wdyt?

I also love that you're pushing the campfire rule!